lib/paginate_firestore.dart

```diff
@@ -252,6 +256,54 @@ class _PaginateFirestoreState extends State<PaginateFirestore> {
 
     return listView;
   }
+
+  Widget _buildPageView(PaginationLoaded loadedState) {
+    var pageView = CustomScrollView(
+      reverse: widget.reverse,
+      shrinkWrap: widget.shrinkWrap,
+      controller: widget.scrollController,
+      scrollDirection: widget.scrollDirection,
+      physics: widget.physics,
+      slivers: <Widget>[
+        if (widget.header != null) SliverToBoxAdapter(child: widget.header),
+        SliverPadding(
+            padding: widget.padding,
+            sliver: SliverFillRemaining(
+              child: PreloadPageView.custom(
+                scrollDirection: Axis.vertical,
```

Comment:
Let's give scroll direction as an option for the user in PaginateFirestore

Reply:
Didn't see this.. will fix.